orchagent/switchorch.h

```diff
@@ -10,6 +10,7 @@
 
 #define SWITCH_CAPABILITY_TABLE_PORT_TPID_CAPABLE                      "PORT_TPID_CAPABLE"
 #define SWITCH_CAPABILITY_TABLE_LAG_TPID_CAPABLE                       "LAG_TPID_CAPABLE"
+#define SWITCH_CAPABILITY_ORDERED_ECMP_CAPABLE                         "ORDERED_ECMP_CAPABLE"
```

Comment:
Should this be named `SWITCH_CAPABILITY_TABLE_ORDERED_ECMP_CAPABLE` for better consistency?

Reply:
updated